tests/test_action_client.py

```diff
@@ -48,7 +48,7 @@ def test_trivial_methods(ac, mocked_responses, name, method):
     """Validate the URL used with trivial requests."""
 
     action_id = "bogus"
-    url = f"https://actions.api.globus.org/{action_id}/{name}"
+    url = f"https://actions.globus.org/{action_id}/{name}"
```

Comment:
Seeing that there's multiple lines changed in this and test_flow_client, I'd consider declaring and using https://actions.globus.org and https://flows.globus.org as a constant base

Reply:
That's a great idea (especially since I just declared the Actions URL as a constant)! I'll modify these to use the constants before merging.

Reply:
I've amended the last commit and force pushed. After CI runs, I'll merge this branch.

Thanks for the quick review, Lei!